cmd/crc/cmd/start.go

```diff
@@ -60,7 +60,7 @@ func runStart(arguments []string) (*machine.StartResult, error) {
 	checkIfNewVersionAvailable(config.Get(cmdConfig.DisableUpdateCheck).AsBool())
 
 	if err := preflight.StartPreflightChecks(config); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("%w\nplease try 'crc setup' first", err)
```

Comment:
Would be best to return a specific error type/an error which can be matched against IsPreflightFailure(err) so that the upper layers can print this correctly
```
if IsPreflightFailure(err) {
    logging.Infof("Preflight checks failed during `crc start`,  please try to run `crc setup` first in case you haven't done so yet")
}
```
(main advantage is that this will get a nice INFO prefix, and won't look like it's part of thenot necessarily user-friendly error message)

Reply:
Here the upper layer is viper RunE method which just put the error in case any. Right now `if err := preflight.StartPreflightChecks(config)` which means the error is from the PreflightChecks. Do you want to modify the `StartPreflightCheck` signature which return the preflight failure specific error and here we just detect it ?

Reply:
What I had in mind was (probably) to make changes in `cmd/crc/cmd/start.go.prettyPrintTo()` using 
```
var PreflightError
if errors.As(err, &e) {
     fmt.Printf("Preflight checks failed during `crc start`,  please try to run `crc setup` first in case you haven't done so yet")
   
} else {
    return ...
}
```
Sticking generic user-level instructions in an error because we know crc code prints errors using `fmt.Printf` at the end of execution seems like something which could get broken anytime, especially with no corresponding integration test.